### PR TITLE
[plugin] Add `php.ini` in the PHP plugin

### PR DESCRIPTION
**Pull request by Lagoja**
## Summary

Adds an editable `php.ini` to the PHP plugin. This is useful when developers need to configure their PHP instance, and by creating it in `devbox.d` we let developers customize it for each of their PHP projects. This addresses #1035 and the issue reported by @jrseliga

## How was it tested?

* On a clean PHP project, run `devbox add php@latest` and verify that the php.ini file is created in devbox.d
* Also check the updated example in this PR

### CODE REVIEW

**Comment by jrseliga** [comment]
Thanks @Lagoja! Looks good to me, the `php.ini` file is published and it seems the settings defined within take precedent.

**Comment by mikeland86** [comment]
@Lagoja does php pick up the php.ini automatically? I thought it had to be in a special dir.

**Comment by Lagoja** [comment]
You can use the `$PHPRC` environment variable (which I added to the plugin) to tell PHP where to look for the php.ini file. It's the second location that PHP will look, after the SAPI module specific location: 

https://www.php.net/manual/en/configuration.file.php

It seems to work on testing -- I can use it to set the xdebug mode, or to bump the memory limit for Django

**Comment by mikeland86** [comment]
missed that, awesome!

File: plugins/php.json [hunk 1]
```diff
@@ -10,10 +10,12 @@
   "env": {
     "PHPFPM_ERROR_LOG_FILE": "{{ .Virtenv }}/php-fpm.log",
     "PHPFPM_PID_FILE": "{{ .Virtenv }}/php-fpm.log",
-    "PHPFPM_PORT": "8082"
+    "PHPFPM_PORT": "8082",
+    "PHPRC": "{{ .DevboxDir }}"
```
Comment (mikeland86) [inline plugins/php.json:14]: Does this break if file/directory are missing? (user might delete `devbox.d/php` and we don't recreate.

**Review by mikeland86 — APPROVED** [review]

**Review by mikeland86 — APPROVED** [review]

File: plugins/php.json (hunk 1 above)
Comment (Lagoja) [reply plugins/php.json:14]: Yes, if no `php.ini` is detected, php will use the default values for all directives.

**Comment by Lagoja** [comment]
Going to tweak the examples to be backwards compatible, and then I'll merge

**Pull request merged**